source/module_hamilt_lcao/hamilt_lcaodft/operator_lcao/test/CMakeLists.txt

```diff
@@ -15,13 +15,40 @@ AddTest(
   SOURCES test_overlapnew_cd.cpp ../overlap_new.cpp ../../../module_hcontainer/func_folding.cpp 
```

Comment:
please change test_overlapnew_cd.cpp to another name like "operator_overlapnew_cd_test.cpp"